client/src/pages/people/Form.js

```diff
@@ -904,6 +930,25 @@ const PersonForm = ({
       }
     }
   }
+  async function checkPotentialDuplicates(firstName, lastName) {
+    if (
+      !forOnboarding &&
+      !edit &&
+      firstName.length >= MIN_CHARS_FOR_DUPLICATES &&
+      lastName.length >= MIN_CHARS_FOR_DUPLICATES
```

Comment:
Since only the last name is required, no duplicate warning will be shown if only the last name has been filled in. Perhaps take the opportunity to improve this as well?